.travis.yml

```diff
@@ -29,6 +29,7 @@ addons:
     sources:
     - trusty-media
     packages:
+    - libgnutls28-dev
```

Comment:
For some reason `pycurl` couldn't be built without installing this package. I'm not sure what changed but it seems something has.

Reply:
see https://github.com/thumbor/thumbor/pull/1207